documentation/modules/security/proc-replacing-your-own-private-keys.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-replacing-your-own-private-keys-{context}']
+= Replacing private keys used by your own CA certificates
+
+This procedure describes how to renew CA certificates and keys you installed yourself, instead of using the certificates generated by the Cluster Operator.
+Follow this procedure when you are changing the private keys used to sign the new CA certificates.
```

Comment:
Same as above, is it changing the private keys used to sign the CA? Or the CA private key? Not the same thing.

Reply:
I just removed the sentence in this case, because the previous one and even the title should be clear about you using a new CA key.